content/how-to/fabric/fabric-ticketing/index.md

```diff
@@ -0,0 +1,153 @@
+
+#### **Left Side Ticket Panel**: 
+
+<img width="302" alt="FT9" src="https://user-images.githubusercontent.com/98875667/156182984-cf86867f-a906-45bd-9e37-e0a8daba6c67.png">
+
+1. Create Ticket: User can create new ticket using this option. Details of how new ticket is created is detailed in upcoming sections. 
+2. Search: User can search the required tickets. 
+
+<img width="142" alt="FT10" src="https://user-images.githubusercontent.com/98875667/156183312-0c15933a-b591-4deb-8dd8-1ccbb2682681.png">
+
+3. Status Drop Down: Allows a user to narrow down the list results by ticket status of Open Tickets, Pending Customer, Closed Tickets, and All Tickets. 
+
+<img width="257" alt="FT11" src="https://user-images.githubusercontent.com/98875667/156183533-ce5b707d-2d71-45ea-b626-2c5c1ec7c9e4.png">
+
+4. Filter: This filter option can filter your tickets using options such as Device, Created By, Accounts, and Date Range 
+5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list. 
+
+#### **Right Side Ticket Panel**:  
+
+**Selected Ticket Panel**: Displays the details of the selected ticket. This loads the first ticket list item by default. 
+
+<img width="621" alt="FT13" src="https://user-images.githubusercontent.com/98875667/156184075-2b34647c-1c4d-4053-b4ea-20486b52b52f.png">
+
+<img width="374" alt="FT8" src="https://user-images.githubusercontent.com/98875667/156186872-21df8151-9e8f-44f2-abb4-c9fca88e1902.png">
+
+You can load your ticket in a new browser tab by selecting the ticket ID from the header of the right ticketing panel as shown in above image.
+
+The right-side ticket panel displays the details of the selected ticket. The details of the ticket are divided into three tabs such as: 
+
+* **Comments**: Comment option displays the details of the most recently created ticket. 
+
+<img width="691" alt="FT14" src="https://user-images.githubusercontent.com/98875667/156187554-fc47f74f-89a0-44b5-865c-ccbb90353123.png">
+
+* **Details**: You can view the details of the tickets in terms of the category, created by, modified by, severity, created date, recipients and so forth.
+
+<img width="290" alt="FT15" src="https://user-images.githubusercontent.com/98875667/156187836-c9be71b9-a9f2-408f-a918-84cb46e277f6.png">
+
+You can **Add Recipient** to the ticket as shown in the above image. The added recipients will get notification about the ticket once added. The **Add Recipient** pop over will hide contacts that have ticket notifications settings disabled on a dedicated account and does not apply to multi-cloud contacts.
+
+<img width="280" alt="FT16" src="https://user-images.githubusercontent.com/98875667/156188912-77a59f2c-d667-4038-afa3-f4c4747a4db1.png">
+
+You can **Add Groups** to the ticket as shown in the above image. The added groups will get notifications about the ticket. This option is available for the dedicated users.
```

Comment:
Remove the sentence `This option is available for the dedicated users.`
Add Groups is now available for both dedicated and non-dedicated users also.

Reply:
Removed